Userland/Libraries/LibWeb/Painting/NestedBrowsingContextPaintable.cpp

```diff
@@ -51,7 +51,7 @@ void NestedBrowsingContextPaintable::paint(PaintContext& context, PaintPhase pha
         auto old_viewport_rect = context.device_viewport_rect();
 
         context.painter().add_clip_rect(clip_rect.to_type<int>());
-        context.painter().translate(absolute_rect.x().value(), absolute_rect.y().value());
+        context.painter().translate(clip_rect.x().value(), clip_rect.y().value());
```

Comment:
Even if the value is correct, it feels wrong to say "clip rect" here, since the high-level logic isn't "translate the painter by the clip rect's location", but rather "translate the painter by the absolute rect location, scaled to device pixels"